src/schedule.py

```diff
@@ -439,7 +439,7 @@ def pre_schedule(computation_graph, hw_netlist):
     for u, v, data in operator_edges:  # this shouldn't run after the first iteration
         if (u, v) not in hw_netlist.edges():
             new_node_name = f"tmp_op_reg;{new_node_id}"
-            computation_graph.add_node(new_node_name, function="Regs")
+            computation_graph.add_node(new_node_name, function="Regs", cost=0.0)
```

Comment:
this should be the cost of a register, might need to pass in the latency dict from tech params to this function